jets/e/ripe.c

```diff
@@ -0,0 +1,80 @@
+/* j/5/ripe.c
+**
+*/
+#include "all.h"
+#include <openssl/evp.h>
+
+u3_noun
+u3qe_ripe(u3_atom wid, u3_atom dat)
+{
+  c3_assert(_(u3a_is_cat(wid)));
+  dat = u3qc_rev(3, wid, dat);
+
+  c3_y* dat_y = (c3_y*)u3a_malloc(wid);  // msg body
+  u3r_bytes(0, wid, (void*)dat_y, dat);
+
+  const EVP_MD* rip_u = EVP_ripemd160(); // ripem algorithm
+  EVP_MD_CTX* con_u = NULL;              // context
+
+  /* build library context object
+     we do this once (and only once)
+  */
```

Comment:
I think `con_u` needs a `static` modifier (as in the `secp256k1_context` elsewhere in this PR). Please add the modifier or remove the comment.